### PR TITLE
Add Storage Domains under Data Center

### PR DESCRIPTION
**Pull request by roniezr**
It seems that storage domain status can be reached only through
data center
This patch add the storage domain belonging to the data center

### CODE REVIEW

**Review by kobihk — APPROVED** [review]

File: ovirtlib4/data_centers.py [hunk 1]
```diff
@@ -27,3 +27,31 @@ class DadaCenterEntity(CollectionEntity):
...
         super().__init__(*args, **kwargs)
+
+    @property
+    def storage_domains(self):
+        return DataCenterStorageDomains(connection=self.service)
+
+
+class DataCenterStorageDomains(CollectionService):
+    """
+    Gives access to all VM NICs
```
Comment (MosheSheena) [inline ovirtlib4/data_centers.py:38]: How are SD related to VM NICs?

File: ovirtlib4/data_centers.py [hunk 2]
```diff
@@ -27,3 +27,31 @@ class DadaCenterEntity(CollectionEntity):
...
+        self.entity_type = types.StorageDomain
+
+    def _get_collection_entity(self):
+        """ Overwrite abstract parent method """
+        return DataCenterStorageDomain(connection=self.connection)
+
+
+class DataCenterStorageDomain(CollectionEntity):
+    """
+    Put StorageDomain custom functions here
```
Comment (MosheSheena) [inline ovirtlib4/data_centers.py:54]: I would replace 'Put' with something more explanatory.
Using a directive in a class doc-string does not feel right to me

**Review by MosheSheena — CHANGES_REQUESTED** [review]

File: ovirtlib4/data_centers.py (hunk 1 above)
Comment (roniezr) [reply ovirtlib4/data_centers.py:38]: yes, it is a copy/paste mistake, I fix it

File: ovirtlib4/data_centers.py (hunk 2 above)
Comment (roniezr) [reply ovirtlib4/data_centers.py:54]: Currently, all doc-string are using this word, so I prefer to keep the convention.

**Comment by roniezr** [comment]
replaced with https://github.com/rhevm-qe-automation/ovirtlib4/pull/13

**Pull request closed without merging**